core/src/main/scala/com/broadinstitute/dsp/package.scala

```diff
@@ -11,10 +13,10 @@ package object dsp {
   val regionName = RegionName("us-central1")
 
   def initGoogleCredentials[F[_]: Sync: Timer](
```

Comment:
Is this method used anywhere?

Reply:
made it used for one site...we can potentially use it in more places